flink-connectors/flink-connector-hive/archunit-violations/26d337fc-45c4-4d03-a84a-6692c37fafbc

```diff
@@ -46,6 +46,12 @@ org.apache.flink.connectors.hive.HiveTableSourceITCase does not satisfy: only on
 * reside in a package 'org.apache.flink.runtime.*' and is annotated with @ExtendWith with class InternalMiniClusterExtension\
 * reside outside of package 'org.apache.flink.runtime.*' and is annotated with @ExtendWith with class MiniClusterExtension\
  or contain any fields that are public, static, and of type MiniClusterWithClientResource and final and annotated with @ClassRule or contain any fields that is of type MiniClusterWithClientResource and public and final and not static and annotated with @Rule
+org.apache.flink.connectors.hive.HiveTableSpeculativeSinkITCase does not satisfy: only one of the following predicates match:
```

Comment:
I think this change is not needed if the test declares `MiniClusterExtension`, see `HiveTableCompactSinkITCase`.